Rx/v2/src/rxcpp/operators/rx-repeat.hpp

```diff
@@ -25,166 +25,46 @@
 #define RXCPP_OPERATORS_RX_REPEAT_HPP
 
 #include "../rx-includes.hpp"
+#include "impl-retry-repeat.hpp"
```

Comment:
The code is the primary documentation :)
If the same feedback is required on several PRs from different authors I would find a way to document that. 
The developer manual was added by a contributor that wanted to record some things that he found helpful, please feel free to expand that with information you think is useful - or put it in code comments - or doxygen comments in the code - or find a better place and structure to store this kind of information.  

Thanks!